frameworks/helloworld/src/main/dist/isolation.yml

```diff
@@ -0,0 +1,16 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{SERVICE_PRINCIPAL}}
+  user: {{SERVICE_USER}}
+pods:
+  hello:
+    count: {{HELLO_COUNT}}
+    isolate-tmp: true
```

Comment:
Make this configurable, deploy with it false, verify no tmp volume, then make it true, and verify there is a tmp volume.